packages/web-runtime/l10n/.tx/config

```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:core]
+[o:owncloud-org:p:owncloud-web:r:web-core]
```

Comment:
Please name this "web-runtime". Don't know why this was core in the past, but the package name is runtime and should be reflected in the resource name.

Reply:
done

Reply:
unresolving, this needs to stay `core`, only the resource_name should be `web-runtime`. Sorry for the confusion.